src/App.jsx

```diff
@@ -14,33 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goodsToSort, setGoodsToSort] = useState(goodsFromServer);
+  const [selectedOption, setSelectedOption] = useState("");
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const optionsToSort = [
+    { type: "Sort alphabetically", className: "is-info", isToAppear: true },
+    { type: "Sort by length", className: "is-success", isToAppear: true },
+    { type: "Reverse", className: "is-warning", isToAppear: true },
+    { type: "Reset", className: "is-danger", isToAppear: !checkIfArrayIsEqualsToFromServer() }
```

Comment:
The `isToAppear` property for the 'Reset' option is set only once during the initial render. This means it won't update dynamically if `goodsToSort` changes. Consider using a state or effect to update this property based on the result of `checkIfArrayIsEqualsToFromServer()`.